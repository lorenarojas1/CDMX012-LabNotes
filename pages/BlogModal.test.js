import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import { create, act } from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import { BlogModal } from './BlogModal';

describe(('Test component Blog'), () => {
  test('render component Blog', () => {
    let blogModal;
    act(() => {
      blogModal = create(
        <Router>
          <BlogModal />
        </Router>,
      );
    });
    expect(blogModal.toJSON()).toMatchSnapshot();
  });
});
