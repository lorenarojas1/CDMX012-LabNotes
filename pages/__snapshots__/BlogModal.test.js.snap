// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component Blog render component Blog 1`] = `null`;

exports[`Test component ModalDelete render component ModalDelete 1`] = `
    <section className="containerPopup">
      <div className="containerExtern">
        <div className="popupBlog">
          <button
            type="button"
            className="btnClosePopup"
        onClick={[Function]}
          >
            X
          </button>
          <p className="titleNote">{[Function]}</p>
          <input
            className="titleBlog"
            type="text"
            placeholder="Title"
            name="tittle"
            value={values.tittle}
        onChange={[Function]}
          />
          <textarea
            className="blogMessage"
            placeholder="Description"
            name="message"
            value={[Function]}
        onChange={[Function]}
          />
          <button
            type="button"
            className="btnSaveBlog"
        onClick={[Function]}
          >
            {[Function]}
          </button>
        </div>
      </div>
    </section>`;
